Driverless AI/disparate-impact-analysis-tutorial/disparate-impact-analysis-tutorial.md

```diff
@@ -0,0 +1,458 @@
+
+After setting the reference level to *1* and the *Disparate impact* variable to **SEX** the following arises: 
+
+The model seems to be performing fairly across all categories, which is good. And such categories are *Adverse Impact*, *Accuracy*, *True Positive Rate*, and *Precision*. 
+
+A reminder *Adverse Impact* refers to the following definition: 
+
+  - Adverse impact is the negative effect an unfair and biased selection procedure has on a protected class. It occurs when a protected group is discriminated against during a selection process, like a hiring or promotion decision.[9]  
+
+  In the US, protected classes include race, sex, age (40 and over), religion, disability status, and veteran status.
+
+Further, *True Positive Rate* refers to the following definition: 
+
+  - In machine learning, the true positive rate, also referred to sensitivity or recall, is used to measure the percentage of actual positives which are correctly identified.[10]
+
+Further we also see a slight population imbalance but not at a worrisome level yet. It will be a problem when the imbalance is huge because it will mean that the model will be learning (through examples) from a particular population or group. 
+
+4. Scroll down and let's take a look at *Group Disparity*.
+
+![0.8-low-threshold](assets/0.8-low-threshold.jpg)
+
+You will notice that your benchmark (1/Male) should be one as it provides the level to compare. In financial services, there is an ‘accepted’ rule of thumb that, as a benchmark, one class should not be treated 80% less favorably than another, as a starting point. This benchmark is up for debate, and therefore, we will see how this will develop going forward, but for now, we have set the low (unfairness) threshold to *.8** (it should already be set to .8). As a result, with a .8 low threshold, we can see that none of the group disparity metrics are less than point 8, which is a good sign.
+
+5. Now let’s adjust the *low threshold* to .9 and see what happens: 
+
+![0.9-low-threshold](assets/0.9-low-threshold.jpg)
+
+We can see that if we adjust the low threshold, the group disparity metrics will become highlighted and begin to flag, saying that if .9 is the cutoff,  this class (2/Women) will be treated unfairly by the model. Note: the *True Positive Rate Disparity* and *False Negative Rate Disparity* are not affected.
+
+6. Let’s scroll down and investigate *Group Parity* and check if all classes are being treated fairly by the model under the benchmark thresholds.
+
+![group-parity](assets/group-parity.jpg)
+
+Here we can see *True* across all classes and metrics, which is what we want to see.
+
+### References
+
+- [9] [WHAT IS ADVERSE IMPACT? AND WHY MEASURING IT MATTERS](https://www.hirevue.com/blog/what-is-adverse-impact-and-why-measuring-it-matters)
+
+- [10] [Encyclopedia of Systems Biology](https://link.springer.com/referenceworkentry/10.1007%2F978-1-4419-9863-7_255)
+
+## Task 5: Sensitivity Analysis Part 1
```

Comment:
Shouldn't this be more like Bias analysis? It kind of confuses me that we are focusing on Bias but the Task name gives the impression we are focusing on Sensitivity Analysis. Consider renaming this and the following task to something like "Finding Bias with Sensitivity Analysis" or something that includes "Bias"

Reply:
My suggestion is based on the title of the tutorial and what seems to be the focus of this tutorial which is to see if your models are fair by not introducing bias. After reading this and the following task, you do focus on bias, so even though you are using the Sensitivity Analysis tool, in my opinion, you should keep your focus on Bias.